src/components/footer/Footer.scss

```
@@ -0,0 +1,245 @@
+          }
+          span:nth-child(2) {
+            font-size: 18px;
+            font-weight: 700;
+            color: #595959;
+            text-decoration: underline;
+            cursor: pointer;
+          }
+          span:nth-child(3) {
+            font-family: 'Noto_M,sans-serif';
+            font-weight: 700;
+            font-size: 13px;
+            color: #797979;
+          }
+          span:nth-child(4) {
+            color: #6b6b6b;
+            font-size: 12px;
+          }
+        }
+      }
+      .right {
+        display: flex;
+        div {
+          height: 78px;
+          margin: 0 25px;
+          cursor: pointer;
+          #one {
+            margin-left: 17px;
+          }
+          #two {
+            margin-left: 22px;
+          }
+          #three {
+            margin-left: -4.5px;
+          }
+          p {
+            margin-top: 10px;
+            color: #8a8a8b;
+            font-size: 16px;
+            font-weight: 600;
+          }
+        }
+      }
+    }
+  }
+  .bottomSec {
+    margin-top: 40px;
+    padding-left: 40px;
+    display: flex;
+    justify-content: center;
+    .bottomSizer {
+      max-width: 1400px;
+      width: 1400px;
+      letter-spacing: -0.05em;
+      @include flexCenter;
+      justify-content: space-between;
+      .spacer {
+        display: flex;
+        .logo {
+          height: 109px;
+          width: 83px;
+          margin-top: 10px;
+        }
+        .left {
+          margin-left: 40px;
+          .topList {
+            ul {
+              width: 750px;
+              display: flex;
+              li {
+                cursor: pointer;
+                @include flexCenter;
+                border-right: solid rgb(121, 119, 119) 1px;
+                height: 10px;
+                margin: 0 2px;
+                padding-right: 5px;
+                font-size: 12px;
+                background-color: white;
+                color: rgb(63, 63, 63);
+                letter-spacing: -0.1em;
+              }
+              li:nth-child(3) {
+                font-weight: bold;
+              }
+              li:last-child {
+                border: none;
+              }
+            }
+          }
+          .info {
+            margin-left: 7px;
+            margin-top: 17px;
+            span {
+              font-size: 12px;
+              color: #a0a0a0;
+            }
+            .top {
+              display: flex;
+              margin: 10px 0;
+              align-items: center;
+              button {
+                margin-left: 10px;
+                width: 120px;
+                height: 32px;
+                line-height: 26px;
+                color: #828282;
+                font-size: 12px;
+                display: flex;
+                justify-content: center;
+                border: 1px solid #ededed;
+                background: #fafafa;
+              }
+              button:focus {
+                outline: none;
+              }
```

```suggestion
              button {
                margin-left: 10px;
                width: 120px;
                height: 32px;
                line-height: 26px;
                color: #828282;
                font-size: 12px;
                display: flex;
                justify-content: center;
                border: 1px solid #ededed;
                background: #fafafa;
              
                &:focus {
                  outline: none;
                }
              }
```
네스팅과 & 기능 활용해주세요!
그리고 css 순서도 적용해주세요!